### ISSUE TITLE
Installing libreadline5-dev on Ubuntu 12.04 fails

### ISSUE THREAD

**Issue by caryp** [miketheman/ruby_enterprise#1]
The package to install should be libreadline-dev.

Here is the failure:

```
[Fri, 18 Jan 2013 23:41:11 +0000] INFO: Processing package[libreadline5-dev] action install (ruby_enterprise::default line 38)
[Fri, 18 Jan 2013 23:41:11 +0000] ERROR: package[libreadline5-dev] (ruby_enterprise::default line 38) has had an error
[Fri, 18 Jan 2013 23:41:11 +0000] ERROR: package[libreadline5-dev] (/tmp/vagrant-chef-1/chef-solo-1/cookbooks/ruby_enterprise/recipes/default.rb:38:in `block in from_file') had an error:
package[libreadline5-dev] (ruby_enterprise::default line 38) had an error: Chef::Exceptions::Package: libreadline5-dev has no candidate in the apt-cache
```

**Comment by miketheman** [comment]
Hi @caryp! Thanks for reporting this problem.

From my initial diagnosis, it looks like Ubuntu 12.04 no longer includes `libreadline5-dev`. Can I ask you to try modifying Line 34 of default.rb to `libreadline-dev` and try running it?  I also know there are others out there with problems on Ubuntu 12.04 - http://awesomism.co.uk/217443

-M

**Comment by miketheman** [comment]
Also, it looks like it's approaching End Of Life soon and has no maintainers: http://blog.phusion.nl/2012/02/21/ruby-enterprise-edition-1-8-7-2012-02-released-end-of-life-imminent/

I've since moved to Ruby 1.9.3 - it has all the speed I need.

**Comment by miketheman** [comment]
If `libreadline-dev` doesn't work, maybe try `libreadline-gplv2-dev` also - there's some crazy name changes Ubuntu made.

**Comment by caryp** [comment]
Oops. Yeah, that was a typo, the package to install should be `libreadline-dev`.  At least it worked for me.

**Comment by miketheman** [comment]
So: historically, the package required was libreadline5-dev - I am considering leaving this as-is for older versions of ubuntu and the fact that REE is EOL, pretty much in favor of 1.9.3 (that has all the perf fixes and such).

**Comment by caryp** [comment]
I agree that ruby 1.9.3 is the way to go.  The problem is getting the time to upgrade our application.  I have created a pull request that fixes this issue for ubuntu 12.04:

https://github.com/miketheman/ruby_enterprise/pull/2

I have tested on both Ubuntu 10.04 and Ubuntu 12.04.
